lib/liquid/tags/include.rb

```diff
@@ -16,18 +16,20 @@ module Liquid
   #   {% include 'product' for products %}
   #
   class Include < Tag
-    Syntax = /(#{QuotedFragment}+)(\s+(?:with|for)\s+(#{QuotedFragment}+))?/o
+    SYNTAX = /(#{QuotedFragment}+)(\s+(?:with|for)\s+(#{QuotedFragment}+))?(\s+(?:as)\s+(#{VariableSignature}+))?/o
```

Comment:
Why `#{VariableSignature}+` and not just `#{VariableSignature}`? (if we expect only one)

Reply:
`VariableSignature` = `/\(?[\w\-\.\[\]]\)?/` only matches characters so we expect many

Reply:
I'm actually thinking now I should change this to `VariableSegment` = `/[\w\-]/`